mubench.reviewsite/tests/SlimTestCase.php

```diff
@@ -177,4 +177,10 @@ protected function createFindingWith($experiment, $detector, $misuse)
         $finding->method = "method(A)";
         $finding->save();
     }
+
+    function createReview($misuse, $reviewer, $hit, $violations = [], $tags = [])
```

Comment:
This does not belong into this base class. Since it's reused between many tests, may place it in a `review_test_utils` or something?